client/components/lists/listHeader.js

```diff
@@ -47,6 +49,70 @@ Template.listActionPopup.events({
   },
 });
 
+Template.listImportCardPopup.events({
+  submit(evt) {
+    // 1. get the json data out of the form and parse it
+    evt.preventDefault();
+    const jsonData = $(evt.currentTarget).find('textarea').val();
+    const data = JSON.parse(jsonData);
```

Comment:
I think it’s worth implementing error handling here before merging the feature. Maybe we could use something like [this](https://github.com/wekan/wekan/blob/758be3356cc3fb05de0a29787059b6b34aac08a2/client/components/users/userAvatar.js#L100-L106)?